README.md

```diff
@@ -93,146 +66,95 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
+<br>
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **key_feature_1**
+- **key_feature_2**
+- **key_feature_3**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+
+
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> Project linked below.
 
-- [Live Demo Link](https://google.com)
+- [Hello-Microverse](https://github.com/joshield/Hello-Microverse)
```

Comment:
As requested by the previous reviewer, since your project is not deployed, please let's consider removing the Live demo link to avoid broken links in our readme. 

We can put a text e.g `Live demo is not available` instead 💪